projects/dev-schematics/src/add-routing/index.ts

```diff
@@ -1,8 +1,20 @@
-import { chain, Rule, Tree } from '@angular-devkit/schematics';
+import { chain, Rule, SchematicsException, Tree } from '@angular-devkit/schematics';
+import { getAppModulePath } from '@schematics/angular/utility/ng-ast-utils';
+import { getProjectTargets } from '@spartacus/schematics';
+import { Schema as SpartacusDevSchematicsOptions } from '../ng-add/schema';
 import { insertPropertyInStorefrontModuleCallExpression } from '../shared/utils/module-file-utils';
 
-function provideTestRouting(): Rule {
+function provideTestRouting(options: SpartacusDevSchematicsOptions): Rule {
```

Comment:
`provideTestRouting` seems misleading, but can't remind myself reason behind this name. Maybe let's change it to more appropriate

Reply:
Sure. Do you have any suggestion?

Reply:
After an offline discussion, we decided to leave it as is.